release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -196,6 +203,16 @@ submodule openconfig-bgp-common-multiprotocol {
         "This leaf indicates whether the AFI-SAFI is enabled for all
         neighbors or groups.";
     }
+
+    leaf send-community {
```

Comment:
This should be `send-community-type`.  Also, similar to #852 ,  should this be a list of types?  Or is only a single type supported?

Reply:
I have made this two separate PRs because I figured they were two different issues: one thing is we wanted to make this configuration available also per address-family and this would be a non-breaking change as it is only a new config. While  #852 would change an existing configuration. But, maybe I can merge both changes in one PR.

Reply:
@dplore Can I close this PR and leave only #852? The changes are included there anyway.

Reply:
Yes you can do this.   I will go ahead and close this one for you for clarity.